cmd/updateprs/updateprs_test.go

```diff
@@ -88,6 +88,82 @@ func TestItDoesNotClosePRsIfNotConfirmed(t *testing.T) {
 	fakeGitHub.AssertCalledWith(t, [][]string{})
 }
 
+func TestItLogsUpdateDescriptionErrorsButContinuesToTryAll(t *testing.T) {
+	fakeGitHub := github.NewAlwaysFailsFakeGitHub()
+	gh = fakeGitHub
+
+	testsupport.PrepareTempCampaign(true, "org/repo1", "org/repo2")
+
+	out, err := runUpdateDescriptionCommandAuto("README.md")
+	assert.NoError(t, err)
+	assert.Contains(t, out, "Updating PR description in org/repo1")
+	assert.Contains(t, out, "Updating PR description in org/repo2")
+	assert.Contains(t, out, "turbolift update-prs completed with errors")
+	assert.Contains(t, out, "2 errored")
+
+	fakeGitHub.AssertCalledWith(t, [][]string{
+		{"work/org/repo1", "PR title", "PR body"},
+		{"work/org/repo2", "PR title", "PR body"},
+	})
+}
+
+func TestItUpdatesDescriptionsSuccessfully(t *testing.T) {
+	fakeGitHub := github.NewAlwaysSucceedsFakeGitHub()
+	gh = fakeGitHub
+
+	testsupport.PrepareTempCampaign(true, "org/repo1", "org/repo2")
+
+	out, err := runUpdateDescriptionCommandAuto("README.md")
+	assert.NoError(t, err)
+	assert.Contains(t, out, "Updating PR description in org/repo1")
+	assert.Contains(t, out, "Updating PR description in org/repo2")
+	assert.Contains(t, out, "turbolift update-prs completed")
+	assert.Contains(t, out, "2 OK, 0 skipped")
+
+	fakeGitHub.AssertCalledWith(t, [][]string{
+		{"work/org/repo1", "PR title", "PR body"},
+		{"work/org/repo2", "PR title", "PR body"},
```

Comment:
Shouldn't this test amend the description? `TestItUpdatesDescriptionsFromAlternativeFile` does verify that we sent a revised title/body to `fakeGitHub`, and I think we should do similarly here.

Reply:
good point!